siloLapisTests/test/aggregated.spec.ts

```diff
@@ -38,7 +38,7 @@ describe('The /aggregated endpoint', () => {
       })
     );
 
-  it('should correctly handle mutliple mutation requests in GET requests', async () => {
+  it('should correctly handle mutliple mutation requests', async () => {
```

Comment:
I think I initially implemented those tests to really send a GET request so that we test that at least in a few cases -> rather make it a GET request than renaming the test.

Reply:
I added a test for this get request, but kept the test for the post request. Should this also be done for the other endpoints?

Reply:
I think I added this so test that the deserialization of arrays in GET requests works. I see those e2e tests that way:
* We test most of the logic via POST requests, since we can conveniently use the generated client for that.
* We send a few GET requests to have at least some e2e tests for those, too.
* In general I would rely on the controller tests in the Kotlin code. The e2e tests are mainly there to make sure that it works from the outside and that the integration with SILO works.

So I think we don't need the tests for both request methods. But having some GET requests for other endpoints doesn't hurt.